### DISCUSSION TITLE
OMC 4 Android

### DISCUSSION THREAD

**Discussion by mariush444** [osmandapp/OsmAnd#20769] [Ideas]
Would it be possbile to recode OMC to run it on Android?
jar can't be started on Android and OMC is quite powerfull to create some staff by user or plugins. Use it locally on smartphon could be greate.
What is your opinions, devs?

**Comment by vshcherb** [comment]
Nope it's not possible, it's too big and has some dependencies to "native" libs like sqlite. It also won't run on Java as Android has RAM limit for java 256-512MB which is far not enough for essential tasks of map generations

**Comment by mariush444** [comment]
Thank you for answer.

**Comment by mariush444** [comment]
Termux with java17 works OK for utilities
